src/rdkafka_transport.c

```diff
@@ -806,6 +945,138 @@ static int rd_kafka_transport_ssl_handshake (rd_kafka_transport_t *rktrans) {
 	return 0;
 }
 
+/**
+* Registers the SSL client certificates on the SSL_CTX
+*
+* Returns -1 on error, 0 if client certificates are successfully registered.
+*/
+int rd_kafka_transport_set_client_certificates(rd_kafka_t *rk,
+    SSL_CTX *ctx,
+    char **errstr, size_t *errstr_size) {
```

Comment:
these two should be passed by the caller, not returned from this function:
change it to `char *errstr, size_t errstr_size`